src/app/shared/pricing-definitions/pricing-definition/pricing-definition.component.ts

```diff
@@ -97,23 +99,21 @@ export class PricingDefinitionComponent implements OnInit {
 
   public ngOnInit(): void {
     // TODO : show current entity name instead of id - for others than charging station c'est pas relevant
-    this.context = this.currentEntityType === Entity.TENANT ? this.centralServerService.getLoggedUser().tenantName : this.currentEntityID;
+    this.context = this.currentEntityType === Entity.TENANT ? this.centralServerService.getLoggedUser().tenantName : this.currentEntityName;
     this.formGroup = new FormGroup({
       id: new FormControl(),
       entityID: new FormControl(this.currentEntityID),
       entityType: new FormControl(this.currentEntityType),
-      // TODO: add restrictions fields
       restrictions: new FormGroup({}),
       name: new FormControl('',
         Validators.compose([
           Validators.required,
-          Validators.maxLength(100),
         ])),
       staticRestrictions: new FormGroup({
         validFrom: new FormControl(null),
         validTo: new FormControl(null),
         connectorPowerEnabled: new FormControl(false),
-        connectorPowerkW: new FormControl(null, Validators.pattern('[0-9]*[,.]?[0-9]{1,2}')),
+        connectorPowerkW: new FormControl(null, Validators.pattern(/^-?((\d+(\.\d+)?))$/)),
```

Comment:
Could be useful to use a constant for the regex ;-)

Reply:
will push the change ✅